s3-import-commons/src/test/java/no/unit/nva/publication/s3imports/FilenameEventEmitterTest.java

```diff
@@ -112,7 +114,45 @@ void shouldEmitEventsWithFullFileUriForEveryFilenameInS3BucketWhenInputIsAnExist
         
         assertThat(fileList, containsInAnyOrder(injectedFiles.toArray(URI[]::new)));
     }
-    
+
+    @Test
+    void shouldThrowExceptionWhenSubtopicIsNotSet() {
+        var importRequest = new EventReference(FILENAME_EMISSION_EVENT_TOPIC,
+                                               EMPTY_SUBTOPIC,
+                                               SOME_S3_LOCATION,
+                                               NOW);
+        Executable action = () -> handler.handleRequest(toJsonStream(importRequest), outputStream, CONTEXT);
+        assertThrows(IllegalArgumentException.class, action);
+    }
+
+    @Test
+    void shouldThrowExceptionWhenSubtopicIsNotRecognized() {
+        var importRequest = new EventReference(FILENAME_EMISSION_EVENT_TOPIC,
+                                               "some.invalid.subtopic",
+                                               SOME_S3_LOCATION,
+                                               NOW);
+        Executable action = () -> handler.handleRequest(toJsonStream(importRequest), outputStream, CONTEXT);
+        assertThrows(IllegalArgumentException.class, action);
+    }
+
+    @ParameterizedTest(
+        name = "should emit supopic from importRequest when valid subtopics are supplied")
```

Comment:
supopic :)

Reply:
Når jeg kaller reformat file, får jeg opp over 30 endringer, har du kjørt reformat på denne?

Reply:
Fikset typo. 

Har ikke kjørt reformat på denne modulen så får masse endringer i codestyle for å matche "UnitStyle".
Fjernet disse for code-review slik at det skulle være lettere å se reelle endringer i koden.